packages/widget-chat/src/actions/notifications.js

```diff
@@ -0,0 +1,75 @@
+export const NOTIFICATION_TYPE_POST = `NOTIFICATION_TYPE_POST`;
```

Comment:
Good call. The actions and reducers you put together look flexible enough that we can add those in pretty easily as those features get developed.